addons/mail/static/src/new/composer/suggestion_service.js

```diff
@@ -21,13 +21,13 @@ class SuggestionService {
         const cleanedSearchTerm = cleanTerm(term);
         switch (delimiter) {
             case "@": {
-                this.fetchPartners(cleanedSearchTerm, thread);
+                await this.fetchPartners(cleanedSearchTerm, thread);
```

Comment:
this RPC will take 10s on our prod the first time, we can't wait for it to display results that we already know in JS